### ISSUE TITLE
Can't deploy python lambdas since pipenv "moved" the lock -r cmd (No such option: -r)

### ISSUE THREAD

**Issue by JoelRoxell** [serverless-stack/sst#1954]
[This line](https://github.com/serverless-stack/sst/blob/master/packages/core/assets/python/Dockerfile.dependencies#L22) will always fail when installing with pipenv 2022.8.14 or later https://github.com/pypa/pipenv/issues/5253

 ```
 Removing intermediate container 033fcf2f073f
 ---> 9e11598b69b4
Step 6/11 : WORKDIR /var/dependencies
 ---> Running in 6051c2fc99c0
Removing intermediate container 6051c2fc99c0
 ---> 86fb0566129d
Step 7/11 : COPY Pipfile* pyproject* poetry* requirements.tx[t] ./
 ---> 9b3574216ea0
Step 8/11 : RUN if [ -f 'Pipfile' ]; then pipenv lock -r > requirements.txt; else echo "Pipfile not found"; fi
 ---> Running in 693298a9987b
Usage: pipenv lock [OPTIONS]
Try 'pipenv lock -h' for help.

Error: No such option: -r
The command '/bin/sh -c if [ -f 'Pipfile' ]; then pipenv lock -r > requirements.txt; else echo "Pipfile not found"; fi' returned a non-zero code: 2

Error: docker exited with status 2
 ```

Solution: Either update the docker cmd or install explicit pipenv version that support `pipenv lock -r > requirements.txt;`

**Comment by itsfabijano** [comment]
We ran into the same issue today. Our workaround was to add custom install commands to the `Function` construct:
```typescript
new sst.Function(this, "HandlerFunction", {
        handler: "...",
        bundle: {
          installCommands: [
            "pipenv requirements > requirements.txt",
            "pip install -r requirements.txt -t .",
          ],
        },
});
```

**Comment by kevinrackspace** [comment]
You can update the core/assets/python/Dockerfile.dependencies and resources/assets/python/Dockerfile.dependencies
line 24 to be
RUN if [ -f 'Pipfile' ]; then pipenv requirements > requirements.txt; else echo "Pipfile not found"; fi

**Comment by kevinrackspace** [comment]
I created a PR to fix this
https://github.com/serverless-stack/sst/pull/1959

**Comment by abrose** [comment]
Hi SST Team, is there any Progress on this issue? The fix seems to be straightforward and there is already a PR for this. 

Regards,
Alfred

**Comment by ssay-work** [comment]
Hello, wanted to follow-up as I am also awaiting a fix for this.

**Comment by fwang** [comment]
Thanks @kevinrackspace! Released in 1.15.3.

^ Had a couple of releases lined up last couple of weeks. Sorry about the delay everyone @JoelRoxell @itsfabijano @abrose @ssay-work
